blocks/columns/columns.js

```diff
@@ -15,4 +15,22 @@ export default function decorate(block) {
       }
     });
   });
+
+  if (block.classList.contains('collections')) {
+    block.parentElement.classList.add('column-collections-parent');
+    // query anchor tags in the block and move the picture into the anchor
+    const anchors = [...block.querySelectorAll('a')];
+    anchors.forEach((anchor) => {
+      const h2 = document.createElement('h2');
+      h2.classList.add('columns-img-header');
+      h2.innerHTML = anchor.textContent.toUpperCase();
```

Comment:
don't assign text to html. `a.innerHMTL = b.innerHTML` is safe, but not tacking plain text and setting that as HTML. 
```suggestion
      h2.textContent = anchor.textContent.toUpperCase();
```

Reply:
Or even better, do the uppercase already in the word document, so `what-you-see-is-what-you-get`. Then you can use `append` instead of `innerHTML` which is cleaner.